app/services/feature_toggle.rb

```diff
@@ -37,4 +37,8 @@ def self.deduplication_flow_enabled?
   def self.duplicate_summary_enabled?
     !Rails.env.production?
   end
+
+  def self.maintenance_mode_enabled?
+    true
```

Comment:
I guess we might want to use an environment variable here instead? That can be set to true or false in the deployment pipelines. Wanted to sanity check the best approach with others before implementing that though.

Reply:
When we enable it, we'll do it by specifying which environments it should be enabled in (like all the other feature toggles).